gpgwrapper/gpgwrapper.go

```diff
@@ -24,6 +26,12 @@ type GnuPG struct {
 	homeDir string
 }
 
+type SecretKeyListing struct {
```

Comment:
Given we're exporting this, shall we comment it in line with the golang style guide?

Reply:
Definitely, good point

Reply:
Done now.